src/NHSD.BuyingCatalogue.Identity.Common/Constants/HealthCheckTags.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.BuyingCatalogue.Identity.Common.Constants
+{
+    public static class HealthCheckTags
+    {
+        public const string Live = "live";
+        public const string Ready = "ready";
```

Comment:
`internal` would be sufficient for the class and constants here.

Reply:
👍

Reply:
Woops, Peter Griffin award to @bjss-chris!

You are of course correct so please ignore my previous comment.